uni/lib/controller/parsers/parser_schedule_html.dart

```diff
@@ -3,12 +3,18 @@ import 'dart:async';
 import 'package:http/http.dart' as http;
 import 'package:html/parser.dart' show parse;
 import 'package:html/dom.dart';
+import 'package:http/http.dart';
+import 'package:uni/controller/networking/network_router.dart';
 import 'package:uni/model/entities/lecture.dart';
 
+import 'package:uni/model/entities/session.dart';
+
 /// Extracts the user's lectures from an HTTP [response] and sorts them by date.
 ///
 /// This function parses the schedule's HTML page.
-Future<List<Lecture>> getScheduleFromHtml(http.Response response) async {
+Future<List<Lecture>> getScheduleFromHtml(http.Response response,
+    Session session) async {
+
```

Comment:
```suggestion
Future<List<Lecture>> getScheduleFromHtml(
    http.Response response, Session session) async {
```
Tip for formatting a file: CTRL-ALT-L